stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/resiliency/failover-connector.bal

```diff
@@ -0,0 +1,248 @@
+package ballerina.net.http.resiliency;
+
+import ballerina.net.http;
+
+
+@Description {value:"Represents Failover connector retry configuration."}
+@Field {value:"failoverCodes: Array of error codes which required retry the requests."}
+@Field {value:"interval: Failover delay interval in millisecond."}
+public struct FailoverConfig {
+    int[] failoverCodes;
+    int interval;
+}
+
+@Description {value:"Represents an error occurred in an action of the Failover connector."}
+@Field {value:"msg: An error message explaining about the error."}
+@Field {value:"cause: The error that caused HttpConnectorError to get thrown."}
+@Field {value:"stackTrace: Represents the invocation stack when FailoverConnectorError is thrown."}
+@Field {value:"statusCode: HTTP status code of the FailoverConnectorError."}
+@Field {value:"httpConnectorError: Array of HttpConnectorError error occurred at each endpoint."}
+public struct FailoverConnectorError {
+    string msg;
+    error cause;
+    StackFrame[] stackTrace;
+    int statusCode;
+    http:HttpConnectorError[] httpConnectorError;
+}
+
+// Represents inferred failover configurations passed to Failover connector.
+struct FailoverInferredConfig {
```

Comment:
Is this meant to be used only within the package? If not, this needs to be public

Reply:
This is only used in this connector.